### PR TITLE
add max_connection_attempts option to prevent infinitely retrying

### PR DESCRIPTION
**Pull request by holyachon**
Hello. I had a problem that aiofcm is retrying infinitely when creating connection was failed (ex. due to invalid api_key, etc..)

I found https://github.com/Fatal1ty/aiofcm/issues/8 this issue, so i added `max_connection_attempts` option similar with https://github.com/Fatal1ty/aioapns/pull/11.

Thank you in advance.

### CODE REVIEW

File: aiofcm/client.py [hunk 1]
```diff
@@ -7,9 +7,9 @@
 
 
 class FCM:
-    def __init__(self, sender_id, api_key, max_connections=10, loop=None):
-        # type: (int, str, int, Optional[asyncio.AbstractEventLoop]) -> NoReturn
-        self.pool = FCMConnectionPool(sender_id, api_key, max_connections, loop)
+    def __init__(self, sender_id, api_key, max_connections=10, max_max_connection_attempts=None, loop=None):
+        # type: (int, str, int, Optional[int], Optional[asyncio.AbstractEventLoop]) -> NoReturn
```
Comment (holyachon) [inline aiofcm/client.py]: Is it okay to leave type hinting style like this?
or do I have to change like
`sender_id: int, api_key: str, max_connections: int = 10, max_max_connection_attempts: Optional[int] = None, loop: Optional[asyncio.AbstractEventLoop] = None` ?

Comment (Fatal1ty) [reply aiofcm/client.py]: The second option would be better!

Comment (holyachon) [reply aiofcm/client.py]: > The second option would be better!

I changed it in [this commit](https://github.com/Fatal1ty/aiofcm/pull/15/commits/47057d2d0c3c5b7e9046909df104eaead8bed61a), and I found there are more methods to be changed.

I opened new [PR](https://github.com/Fatal1ty/aiofcm/pull/16).

**Comment by Fatal1ty** [comment]
Looks good, I'm merging this. Thank you!

File: aiofcm/client.py (hunk 1 above)
Comment (Fatal1ty) [reply aiofcm/client.py]: You're awesome! Type hinting is definitely essential.

**Pull request merged**